cpu/cc2538/dev/scb.h

```diff
@@ -74,6 +74,18 @@
 #define SCB_SYSCTRL_SLEEPEXIT  0x00000002  /**< Sleep on ISR exit */
 /** @} */
 /*---------------------------------------------------------------------------*/
+/** \name SCB_INTCTRL / NVIC_INT_CTRL register bits
+ * @{
+ */
+#define SCB_INTCTRL_NMI_SET   0x80000000   /**< NMI Set Pending */
+#define SCB_INTCTRL_PEND_SV   0x10000000   /**< PendSV Set Pending */
+#define SCB_INTCTRL_UNPEND_SV 0x08000000   /**< PendSV Clear Pending */
+#define SCB_INTCTRL_PENDSTSET 0x04000000   /**< SysTick Set Pending */
+#define SCB_INTCTRL_PENDSTCLR 0x02000000   /**< SysTick Clear Pending */
+#define SCB_INTCTRL_ISR_PRE   0x00800000   /**< Debug Interrupt Handling */
+#define SCB_INTCTRL_ISR_PEND  0x00400000   /**< Interrupt Pending */
+/** @} */
+/*---------------------------------------------------------------------------*/
 
 #endif /* SCB_H_ */
```

Comment:
These definitions are not CC2538-specific, so this file should be moved to `cpu/arm/cortex-m3/` in a separate commit, preferably before this one, and it should be changed to use the official ARM naming from http://infocenter.arm.com/help/topic/com.arm.doc.dui0552a/CIHFDJCA.html.